### ISSUE TITLE
Switch to Pipenv

### ISSUE THREAD

**Issue by 0rzech** [szymonchaber/group-shopping-backend#3]
[Pipenv](https://github.com/pypa/pipenv) is a tool to manage virtualenvs and dependencies. I think it's definitely worth switching to. It is even [officially supported by Heroku](https://github.com/heroku/heroku-buildpack-python) itself.

**Comment by szymonchaber** [comment]
Good idea! I think no preparation is needed on Heroku side, so we can just make a switch.
